### PR TITLE
Add nuxt plugin

### PR DESCRIPTION
**Pull request by ArStah**

### CODE REVIEW

File: nuxt/index.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  Nuxt.js module for bootstrap-vue
+  Usage:
+    - Install semantic-ui-vue and semantic-ui-css packages
+    - Add this into your nuxt.config.js file:
+    {
+      modules: [
+        'semantic-ui-vue/nuxt'
+      ]
+    }
+*/
```
Comment (mariolamacchia) [inline nuxt/index.js]: Can we move this to the readme instead?

File: nuxt/index.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+/*
+  Nuxt.js module for bootstrap-vue
```
Comment (mariolamacchia) [inline nuxt/index.js]: `Nuxt.js module for `~`bootstrap-vue`~`semantic-ui-vue`

Comment (ArStah) [reply nuxt/index.js]: Oops, yes, i just took a code from bootstrap-vue and didnt noticed that)

File: nuxt/index.js (hunk 1 above)
Comment (ArStah) [reply nuxt/index.js]: Yes, i good idea

**Pull request merged**